tests/test_connection.py

```diff
@@ -1194,4 +1194,14 @@ def test_dataloss_protection(node_factory):
     l2.daemon.wait_for_log("Cannot broadcast our commitment tx: they have a future one")
     assert not l2.daemon.is_in_log('sendrawtx exit 0')
 
-    # FIXME: l2 should still be able to collect onchain.
+    closetxid = only_one(bitcoind.rpc.getrawmempool(False))
```

Comment:
On slow systems this may be flaky (since Travis is slow), better to wait for `len(bitcoind.rpc.getrawmempool(False)) == 1`.

Reply:
I've yet to see that actually happen; we had some false positives where it seemed bitcoind accepted a tx which wasn't in the mempool, but it turned out to be a bug in our tests.  If it happens, I'll fix...